app/src/ui/about/about.tsx

```diff
@@ -94,6 +95,17 @@ export class About extends React.Component<IAboutProps, IAboutState> {
       return null
     }
 
+    if (__LINUX__) {
+      const linuxReleaseLink = (
+        <LinkButton uri={LinuxReleasesUri}>View Linux Releases</LinkButton>
```

Comment:
I think we can drop this word to keep it simple:

```suggestion
        <LinkButton uri={LinuxReleasesUri}>View Releases</LinkButton>
```

Reply:
@shiftkey I've updated this in the new commit - thanks!